content/en/blog/_posts/2022-03-31-Kubernetes-1-24-deprecations-and-removals

```diff
@@ -0,0 +1,121 @@
+is Dockershim. Dockershim was deprecated in 1.20. As noted in the [Kubernetes 1.20 changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#deprecation): 
+"Docker support in the kubelet is now deprecated and will be removed in a future release. The kubelet 
+uses a module called "dockershim" which implements CRI support for Docker and it has seen maintenance 
+issues in the Kubernetes community." With the upcoming release of Kubernetes 1.24, the Dockershim will 
+finally be removed. 
+
+In the article [Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/),
+the authors succinctly captured the change's impact and encouraged users to remain calm: 
+> Docker as an underlying runtime is being deprecated in favor of runtimes that use the
+> Container Runtime Interface (CRI) created for Kubernetes. Docker-produced images
+> will continue to work in your cluster with all runtimes, as they always have.
+
+Several guides have been created with helpful information about migrating from dockershim
+to container runtimes that are directly compatible with Kubernetes. You can find them on the
+[Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/)
+page in the Kubernetes documentation.
+
+For more information about why Kubernetes is moving away from dockershim, check out the aptly 
+named: [Kubernetes is Moving on From Dockershim](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/) 
+and the [updated dockershim removal FAQ](/blog/2022/02/17/dockershim-faq/).
+
+Take a look at the [What you need to do to have a working cluster for 1.24](/blog/2022/03/31/ready-for-dockershim-removal/) post to learn about how to ensure your cluster continues to work after upgrading from 1.23 to 1.24. 
+
+## The Kubernetes API removal and deprecation process
+
+Kubernetes contains a large number of components that evolve over time. In some cases, this 
+evolution results in APIs, flags, or entire features, being removed. To prevent users from facing 
+breaking changes, Kubernetes adopted a feature [deprecation policy](/docs/reference/using-api/deprecation-policy/). 
+This policy ensures that stable APIs may only be deprecated when a newer stable version of that 
+same API is available and that APIs have a minimum lifetime depending on their stability level: 
+
+* Generally available (GA) or stable API versions may be marked as deprecated but must not be removed within a major version of Kubernetes. 
+* Beta or pre-release API versions must be supported for 3 releases after deprecation. 
+* Alpha or experimental API versions may be removed in any release without prior deprecation notice. 
+
+Kubernetes will continue to remove APIs by following the deprecation policy regardless of whether an 
+API is removed due to a beta feature graduating to stable or because that API was not proven to be 
+successful and will also make sure migration options are documented. 
```

Comment:
I had trouble parsing the last part of that sentence. Consider rewording. Maybe something like:
"Kubernetes will continue to remove APIs and make sure migration options are documented. It will do this by following the deprecation policy, regardless of whether an API is removed due to a beta feature graduating to stable or because that API was not proven to be  successful."